### ISSUE TITLE
Vaughnlive plugin response length validation

### ISSUE THREAD

**Issue by fragtion** [chrippa/livestreamer#1370]
There seems to be an issue with the Vaughnlive plugin

Similar problem was described at #897 / #904 , but the recommended fix no longer works. 

$ livestreamer http://vaughnlive.tv/stream best
[cli][info] Found matching plugin vaughnlive for URL http://vaughnlive.tv/stream
error: Unable to validate response text: Minimum length is 2 but value is 1
$

Unfortunately I am not very adept with python at this stage.. :( Pls help! :)

**Comment by fragtion** [comment]
Ok some progress:

`DOMAIN_MAP = {
    "breakers": "btv",
    "vapers": "vtv",
    "vaughnlive": "mvn",
}
`

Patch:
-"vaughnlive": "mvn",
+"vaughnlive": "mvn-live",

Still problematic, however:

[cli][error] Failed to read data from stream: Read timeout

**Comment by intact** [comment]
Change "mvn" back to "live" and change INFO_URL to:

```
INFO_URL = "http://mvn.vaughnsoft.net/video/edge/mvn-{domain}_{channel}"
```

**Comment by fragtion** [comment]
That worked. Brilliant, thanks!

**Comment by estimablesir** [comment]
I apologize, but i'm not following how to apply this particular solution to a different channel on vaughnlive.  

I receive the same "error: Unable to validate response text: Minimum length is 2 but value is 1"

originally "livestreamer https://vaughnlive.tv/newzviewz best" was correct, but I now receive the error message above.  Could someone provide me with some guidance?

**Comment by fragtion** [comment]
Yes, sure. Until it gets patched ( dont know how to commit ) , here's a pastebin: http://pastebin.com/jpRuaqEv

File to edit is vaughnlive.py -- in my case this was at /usr/local/lib/python2.7/dist-packages/livestreamer/plugins/vaughnlive.py

**Comment by cloud90** [comment]
i try the script on pastebin but the error dont change! :(

**Comment by fragtion** [comment]
For the record, I'm also using it to watch the newzviewz channel, and it works great for me since making the changes. Make sure you patch all instances of "vaughnlive.py" found on your system, there might be another copy of the file hiding in your local user libs or something. You can also try delete the vaughnlive.pyc and vaughnlive.pyo so that it recompiles them from the source file

**Comment by fragtion** [comment]
Update: looks like they changed the "mvn" to "mnv", causing it to stop working again. Seems a bit like a cat and mouse chase going on? Perhaps the addon script could be made a bit more resilient to the dynamics of these changes

**Comment by cloud90** [comment]
i'm confused , please you cold put the full script? or update the script on pastebin? thank you :)

**Comment by intact** [comment]
@cloud90 https://raw.githubusercontent.com/intact/livestreamer/fix-vaughn/src/livestreamer/plugins/vaughnlive.py

**Comment by cloud90** [comment]
ok work perfectly! :)  thank you intact! :D
